src/lib/button/button.scss

```diff
@@ -52,12 +52,16 @@
 
 // The text and icon should be vertical aligned inside a button
 .mat-button, .mat-raised-button, .mat-icon-button, .mat-fab, .mat-mini-fab {
-  color: currentColor;
   .mat-button-wrapper > * {
     vertical-align: middle;
   }
 }
 
+// Buttons that don't have a background color can inherit the color from its parent.
+.mat-button, .mat-icon-button {
+  color: inherit;
```

Comment:
`inherit` or `currentColor`? Is `inherit` supported on IE11?

Reply:
Yeah, just confirmed that `inherited` is supported.